src/Context/Modal/ModalProvider.jsx

```diff
@@ -25,6 +26,8 @@ const ModalProvider = ({ children }) => {
         setShowAlert,
         showNavMenu,
         setShowNavMenu,
+        showPlaylistModal,
```

Comment:
Context name should be pascalcase. Here it should be ModalContext and not modalContext

Reply:
Noted.